packages/blade/src/components/Spinner/BaseSpinner/BaseSpinner.tsx

```diff
@@ -58,7 +58,7 @@ const getColor = ({
   } else if (contrast == 'low') {
     return getIn(theme.colors, 'brand.gray.700.lowContrast');
   } else {
-    return getIn(theme.colors, 'brand.gray.400.lowContrast');
+    return getIn(theme.colors, 'brand.gray.700.lowContrast');
```

Comment:
both low and high contrast are now same? 🤔 
<img width="399" alt="Screenshot 2023-01-05 at 1 00 13 PM" src="https://user-images.githubusercontent.com/11384858/210725511-b516ef78-f823-44b8-824d-4c2a6ae348db.png">

Reply:
Yeah that's what @saurav12 changed. Maybe we can confirm with him once.

Reply:
@anuraghazra @kamleshchandnani 
On `Payment - Light` it looks different
![image](https://user-images.githubusercontent.com/4298450/210766185-c49ad1d3-1905-4906-ae3d-d5e60fe27584.png)

But on `Banking - Dark` it looks almost similar (not exactly the same) which is expected
![image](https://user-images.githubusercontent.com/4298450/210766302-3a5c75fa-77df-4724-be5a-b29f51de019b.png)

Reply:
but the screenshot i shared is of payment light

Reply:
It might be the same issue as which we have for BaseInput. we need to manually check / update the global color tokens on code. @kamleshchandnani

Reply:
@saurav12 @kamleshchandnani is this looking good? 

<img width="151" alt="image" src="https://user-images.githubusercontent.com/35374649/212054835-27a04834-3c6e-4e22-b333-68b031056bd4.png">